stl/inc/__msvc_system_error_abi.hpp

```diff
@@ -0,0 +1,43 @@
+// __msvc_system_error_abi.h internal header (core)
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef __MSVC_SYSTEM_ERROR_ABI_HPP
+#define __MSVC_SYSTEM_ERROR_ABI_HPP
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+#include <stddef.h>
+
+#pragma pack(push, _CRT_PACKING)
+#pragma warning(push, _STL_WARNING_LEVEL)
+#pragma warning(disable : _STL_DISABLED_WARNINGS)
+_STL_DISABLE_CLANG_WARNINGS
+#pragma push_macro("new")
+#undef new
+
+#ifdef _M_CEE_PURE
+#define __CLRCALL_PURE_OR_STDCALL __clrcall
+#else
+#define __CLRCALL_PURE_OR_STDCALL __stdcall
+#endif

```

Comment:
I was under the impression that for these exactly-5-line-on-off ones we were relaxing that rule. (Note that I copy pasted this from `__CLRCALL_PURE_OR_CDECL` which is why there was the fat finger typo where it still said `__cdecl` before)

Reply:
I'll accept relaxing the rule here, since we have #351 tracking the overall consistency, and there is no real potential for harm (no surrounding nesting).

Reply:
No change requested, resolving.